doc/developing.rst

```diff
@@ -12,50 +12,41 @@ Install Morepath for development
 
 .. highlight:: sh
 
-First make sure you have virtualenv_ installed for Python 3.5.
+Clone Morepath from github and go to the morepath directory::
 
-.. _virtualenv: https://pypi.python.org/pypi/virtualenv
+  $ git clone git@github.com:morepath/morepath.git
```

Comment:
If you are using the dollar sign to mark that it is a shell line the highlight on line 13 should be

```
 .. highlight:: console
```

``sh`` is used for shell scripts or things that look like them.